src/utils.js

```diff
@@ -0,0 +1,94 @@
+const fs = require("fs");
+const child_process = require("child_process");
+const path = require("path");
+const glob = require("glob");
+const semver = require("semver");
+
+function getTags() {
+    return child_process.execSync(`git tag`, {encoding: "utf8"}).split("\n").filter(Boolean);
```

Comment:
I find these chains easier to read if they are lined up by the dots.

```
    return child_process
        .execSync...
        .split...
        .filter...
```

YMMV